Vagrantfile

```diff
@@ -22,8 +22,9 @@ Vagrant.require_version ">= 2.0.0"
 if ARGV.first == "up" && ENV['CILIUM_SCRIPT'] != 'true'
     raise Vagrant::Errors::VagrantError.new, <<END
 Calling 'vagrant up' directly is not supported.  Instead, please run the following:
-  export NWORKERS=n
-  ./contrib/vagrant/start.sh
+$ ./contrib/vagrant/start.sh
```

Comment:
```suggestion
$ NWORKERS=n ./contrib/vagrant/start.sh
```

Reply:
I did read the commit description. 

>First, users tend to forget they export environment variables, so at
least for new contributors, it's probably best to pass the environment
variables in the start.sh command. 

I interpreted this comment as "don't suggest users to export the variable, rather pass it to the start.sh command as an argument.

Reply:
I was referring to the sentence after that:
> Second, the above configuration doesn't make much sense: starting *several* Runtime (i.e., no Kubernetes) VMs is almost never useful.

NWORKERS=n without any other option doesn't make sense to me.

Reply:
Not sure which options you were referring to.  

From the guide - 

>NWORKERS=n: Number of child nodes you want to start with the master, default 0.

This will deploy a single-node cluster by default. Doesn't hurt to mention this briefly along with a link to the guide.

Reply:
Are you suggesting we set it to its default value?

Reply:
I'm suggesting something along these lines - 

```
please run the following to deploy a single-node cluster:
./contrib/vagrant/start.sh

See the development guide for available configuration options.
https://docs.cilium.io/en/latest/contributing/development/dev_setup/

```

Reply:
:ok_hand: